Makefile

```diff
@@ -0,0 +1,10 @@
+all: test run
+
+run:
```

Comment:
[SUGGESTION] If I were you, I would add phony targets.
https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html